### ISSUE TITLE
Travis fails on changes about the Website

### ISSUE THREAD

**Issue by eolivelli** [apache/bookkeeper#2252]
Travis builds are failing due to some error on "Bundler".

```

Warning: the lockfile is being updated to Bundler 2, after which you will be unable to return to Bundler 1.

Bundle complete! 3 Gemfile dependencies, 29 gems now installed.

Bundled gems are installed into `./vendor/bundle`

rm -rf _site local-generated generated_site

JEKYLL_ENV=production bundle exec jekyll build \

	--config _config.yml,_config.apache.yml

Bundler::LockfileError: You must use Bundler 2 or greater with this lockfile.

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/lockfile_parser.rb:108:in `warn_for_outdated_bundler_version'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/lockfile_parser.rb:95:in `initialize'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/definition.rb:72:in `new'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/definition.rb:72:in `initialize'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/dsl.rb:200:in `new'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/dsl.rb:200:in `to_definition'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/dsl.rb:12:in `evaluate'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/definition.rb:33:in `build'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler.rb:129:in `definition'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler.rb:95:in `setup'

  /usr/local/lib/ruby/gems/2.4.0/gems/bundler-1.15.4/lib/bundler/setup.rb:19:in `<top (required)>'

  /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'

  /usr/local/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'

bundler: failed to load command: jekyll (/home/travis/build/apache/bookkeeper/site/vendor/bundle/ruby/2.4.0/bin/jekyll)

make: *** [apache] Error 1

Makefile:24: recipe for target 'apache' failed
```

**Comment by eolivelli** [comment]
@sijie @jiazhai please take a look.
If you have some suggestion I can work on this problem but I don't know about Bundler/Ruby/Jekill

**Comment by aluccaroni** [comment]
Please note that to build the site on my machine (RHEL6) I had to force to use rubygems < 3.0.0 (like 2.7.10) since the makefile is not compatible with >3.0.0 (the no-ri and no-rdoc option for bundler are no more supported). 
Anyway, I don't think this is the problem

**Comment by eolivelli** [comment]
It looks like this problem
https://github.com/jekyll/jekyll/issues/7463

**Comment by eolivelli** [comment]
It looks like it is a problem with the Makefile

if I run directly this command, inside the docker container I have not problem
`bundle exec jekyll build --config _config.yml`

**Comment by eolivelli** [comment]
closed by #2256 thank you @aluccaroni
